samples/java/getting-started/simpleapp/src/main/java/com/ms/samples/App.java

```diff
@@ -0,0 +1,13 @@
+package com.ms.samples;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+        System.out.println( "Hello World!");
```

Comment:
This app is not a web server and doesn't run as a service. It isn't very useful to have this as an example in dev spaces.